src/services/WolfDen.Application/DTOs/Attendence/DailyAttendanceDTO.cs

```diff
@@ -8,7 +8,7 @@ public class DailyAttendanceDTO
         public DateTimeOffset DepartureTime { get; set; }
         public int InsideHours { get; set; }
         public int OutsideHours { get;set; }
-        public string MissedPunch { get; set; }
+        public string? MissedPunch { get; set; }

```

Comment:
What is this value? Why string?

Reply:
misssed punch is string including all the missed punches for the day. it is  nullable since all days wont have missedpunches